src/Modules/Workspaces/Commands.Workspaces/SetPowerBIWorkspace.cs

```diff
@@ -0,0 +1,67 @@
+﻿/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+using System;
+using System.Management.Automation;
+using Microsoft.PowerBI.Api.V2;
+using Microsoft.PowerBI.Api.V2.Models;
+using Microsoft.PowerBI.Commands.Common;
+using Microsoft.PowerBI.Common.Abstractions;
+using Microsoft.PowerBI.Common.Abstractions.Interfaces;
+using Microsoft.Rest;
+
+namespace Microsoft.PowerBI.Commands.Workspaces
+{
+    [Cmdlet(CmdletVerb, CmdletName)]
+    public class SetPowerBIWorkspace : PowerBICmdlet, IUserScope
+    {
+        public const string CmdletName = "PowerBIWorkspace";
+        public const string CmdletVerb = VerbsCommon.Set;
+
+        #region Parameters
+
+        [Parameter(Mandatory = false)]
+        public PowerBIUserScope Scope { get; set; } = PowerBIUserScope.Individual;
+
+        [Parameter(Mandatory = true)]
+        public Guid Id { get; set; }
+
+        [Parameter(Mandatory = false)]
+        public string Name { get; set; }
+
+        [Parameter(Mandatory = false)]
+        public string Description { get; set; }
+
```

Comment:
We should support passing in a Group as a parameter too. We should define ParameterSets